.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: release
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      tag:
+        description: 'tag and release to create'
+        # Input has to be provided for the workflow to run
+        required: true
+env:
+  GITHUB_ENV: ".env"
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - name: run make bump-version
+        run: make bump-version
+        env:
+          NEW_VERSION: ${{ github.event.inputs.tag }}
+      - name: commit manifests
+        uses: EndBug/add-and-commit@a988073222b8bd50f3ecfca9c0ab7dfbf0d08ceb
+        with:
+          message: "create release manifests for ${{ github.event.inputs.tag }}"
+          add: "releases/"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # target commit should be the commit with the new release manifest, not the commit when workflow was triggered
```

Comment:
I'm not sure I understand this one. 🤔  What's the "new release manifest"?

Reply:
Let's say that XXX is the HEAD on master when the workflow was triggered. After bumping the versions, we now have YYY as the HEAD commit. We need to tag the new version at YYY. So, we need to find that target commit sha to pass to the release action (for both softprops/action-gh-release and actions/create-release)

Reply:
Thanks, the explanation made it clear for me now. 👍